addons/web_unsplash/models/__init__.py

```diff
@@ -3,3 +3,4 @@
 
 from . import res_config_settings
 from . import res_users
+from . import ir_qweb
```

Comment:
Missing new line but other than that I just tried and I could add the unsplash image as expected, works like a charm.